io.github.polypixeldev.Polypass.yml

```diff
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: io.github.polypixeldev.Polypass
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: polypass
+separate-locales: false
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --filesystem=xdg-documents
+cleanup:
+  - /git_repo
+  - /io.github.polypixeldev.Polypass.desktop
```

Comment:
this is not what you think it does. It's not "make clean". it's to remove installed files that are not needed.